spec/configuration_spec.rb

```diff
@@ -138,7 +138,7 @@
 
       context "when passed an invalid value" do
         it "should raise an ArgumentError" do
-          expect(lambda{MoneyRails.rounding_mode = "booyakasha"}).to raise_error(ArgumentError, 'booyakasha is not a valid rounding mode')
+          expect { MoneyRails.rounding_mode = "booyakasha" }.to raise_error(ArgumentError, 'booyakasha is not a valid rounding mode')
```

Comment:
I noticed this was also fixed in this PR! https://github.com/RubyMoney/money-rails/pull/705. It's just a deprecation warning 😊